### PR TITLE
added null check for chrome

### PR DESCRIPTION
**Pull request by edewit**
fixes: #3316

## Checklist:

- [ ] Code has been tested locally by PR requester
- [ ] User-visible strings are using the react-i18next framework (useTranslation)
- [ ] Help has been implemented
- [ ] axe report has been run and resulting a11y issues have been resolved
- [ ] Unit tests have been created/updated

### CODE REVIEW

File: apps/admin-ui/src/components/json-file-upload/patternfly/FileUpload.tsx [hunk 1]
```diff
@@ -117,7 +117,8 @@ export const FileUpload: React.FunctionComponent<FileUploadProps> = ({
   const onDropAccepted = (acceptedFiles: File[], event: DropEvent) => {
     if (acceptedFiles.length > 0) {
       const fileHandle = acceptedFiles[0];
-      if (event.type === "drop") {
+      // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+      if (event && event.type === "drop") {
```
Comment (jonkoops) [inline apps/admin-ui/src/components/json-file-upload/patternfly/FileUpload.tsx:121]: Perhaps we can use optional-chaining here?
```suggestion
      if (event?.type === "drop") {
```

File: apps/admin-ui/src/components/json-file-upload/patternfly/FileUpload.tsx [hunk 2]
```diff
@@ -117,7 +117,8 @@ export const FileUpload: React.FunctionComponent<FileUploadProps> = ({
   const onDropAccepted = (acceptedFiles: File[], event: DropEvent) => {
     if (acceptedFiles.length > 0) {
       const fileHandle = acceptedFiles[0];
-      if (event.type === "drop") {
+      // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
```
Comment (jonkoops) [inline apps/admin-ui/src/components/json-file-upload/patternfly/FileUpload.tsx:120]: We might want to add a comment telling future readers of this code that this only breaks in Chrome. Even a link to the issue would be nice.

**Review by jonkoops — APPROVED** [review]

File: apps/admin-ui/src/components/json-file-upload/patternfly/FileUpload.tsx (hunk 2 above)
Comment (edewit) [reply apps/admin-ui/src/components/json-file-upload/patternfly/FileUpload.tsx:120]: it's not our code it's copied from patternfly we don't need to fix it it's temporary here until the issues have been fixed there.

Comment (jonkoops) [reply apps/admin-ui/src/components/json-file-upload/patternfly/FileUpload.tsx:120]: Ahhh, yeah. Missed that.

**Pull request merged**